child.go

```diff
@@ -82,8 +82,8 @@ func (c *child) String() string {
 	return c.proc.String()
 }
 
-func (c *child) Kill() {
-	c.proc.Signal(os.Kill)
+func (c *child) Kill() error {
```

Comment:
That's not a backward compatible change

Reply:
This is probably fine and we can have a minor version bump for this. Caller won't fail but would need to fix lint error if it has linter.